yt/visualization/plot_window.py

```diff
@@ -992,22 +992,19 @@ def _setup_plots(self):
                 if zlim != (None, None):
                     pass
                 elif np.nanmax(image) == np.nanmin(image):
-                    msg = (
-                        "Plot image for field %s has zero dynamic "
-                        "range. Min = Max = %f." % (f, np.nanmax(image))
-                    )
+                    msg = "Plotting %s: All values = %f" % (np.nanmax(image), f)
                 elif np.nanmax(image) <= 0:
-                    msg = (
-                        "Plot image for field %s has no positive "
-                        "values.  Max = %f." % (f, np.nanmax(image))
+                    msg = "Plotting %s: All negative values. Max = %f." % (
+                        f,
+                        np.nanmax(image),
                     )
+                    use_symlog = True
                 elif not np.any(np.isfinite(image)):
-                    msg = f"Plot image for field {f} is filled with NaNs."
```

Comment:
I see you had to rewrite this, but is there a reason that you converted it away from an f-string?

Reply:
My mistake.  Sorry, @jzuhone .  I was just trying to shrink them all so each fit on a single line.  I can take these changes out if people are against it though.  I'll change it back to an f-string.  I think black is going to force them all to be f-strings.

Reply:
No worries!